static/app/views/settings/project/server-side-sampling/modals/uniformRateModal.tsx

```diff
@@ -63,11 +64,13 @@ function UniformRateModal({
 
   const loading = loading30d || !projectStats;
 
-  // TODO(sampling): fetch from API
-  const affectedProjects = ['ProjectA', 'ProjectB', 'ProjectC'];
+  const affectedProjects = recommendedSdkUpgrades.map(
+    recommendedSdkUpgrade => recommendedSdkUpgrade.project.slug
+  );
+
   const [activeStep, setActiveStep] = useState<Step>(Step.SET_UNIFORM_SAMPLE_RATE);
 
-  const uniformSampleRate = rules.find(isUniformRule)?.sampleRate;
+  const uniformSampleRate = uniformRule?.sampleRate;
```

Comment:
```suggestion
  const uniformSampleRate = uniformRule?.sampleRate ?? rules.find(isUniformRule)?.sampleRate;
```
Otherwise this breaks for get started flow

Reply:
ignore my comment